src/binary-encoding.adoc

```diff
@@ -948,76 +955,79 @@ that indicates month (`0`) or day (`1`) precision.
 === Strings
 
 If the high nibble of the opcode is `0x8_`, it represents a string. The low nibble of the opcode
-indicates how many UTF-8 bytes follow. Opcode 0x80 represents a string with empty text ("").
+indicates how many UTF-8 bytes follow. Opcode `0x80` represents a string with empty text (`""`).
 
 Strings longer than 15 bytes can be encoded with the `F8` opcode, which takes a <<varuint, `VarUInt`>>-encoded length
 after the opcode.
 
-==== Example encoding of the empty string (`""`)
-[source]
+.Figure {counter:figure}: Encoding of the empty string, `_""_`
+[source,%unbreakable]
 ----
 ┌──── Opcode in range 80-8F indicates a string
 │┌─── Low nibble 0 indicates that no UTF-8 bytes follow
 80
 ----
 
-==== Example encoding of a 14-byte string
-[source]
+.Figure {counter:figure}: Encoding of a 14-byte string
+[source,%unbreakable]
 ----
 ┌──── Opcode in range 80-8F indicates a string
 │┌─── Low nibble E indicates that 14 UTF-8 bytes follow
 ││  f  o  u  r  t  e  e  n     b  y  t  e  s
-8E 66 6f 75 72 74 65 65 6e 20 62 79 74 65 73
+8E 66 6F 75 72 74 65 65 6E 20 62 79 74 65 73
    └──────────────────┬────────────────────┘
                  UTF-8 bytes
 ----
 
-==== Example encoding of a 24-byte string
-[source]
+.Figure {counter:figure}: Encoding of a 24-byte string
+[source,%unbreakable]
 ----
 ┌──── Opcode F8 indicates a variable-length string
 │  ┌─── Length: VarUInt 24
 │  │   v  a  r  i  a  b  l  e     l  e  n  g  t  h     e  n  c  o  d  i  n  g
-F8 31 76 61 72 69 61 62 6c 65 20 6c 65 6e 67 74 68 20 65 6e 63 6f 64 69 6e 67
+F8 31 76 61 72 69 61 62 6C 65 20 6C 65 6E 67 74 68 20 65 6E 63 6f 64 69 6E 67
       └────────────────────────────────┬────────────────────────────────────┘
                                   UTF-8 bytes
 ----
 
 [[symbols_with_inline_text]]
-=== Symbols with inline text
+=== Symbols With Inline Text
 
-==== Example encoding of a symbol with empty text (`''`)
-[source]
+If the high nibble of the opcode is `0x9_`, it represents a symbol whose text follows the opcode. The low nibble of the
+opcode indicates how many UTF-8 bytes follow. Opcode `0x90` represents a symbol with empty text (`''`).
+
+.Figure {counter:figure}: Encoding of a symbol with empty text (`_''_`)
```

Comment:
We've got seemingly inconsistent typesetting conventions; I don't understand why the `''` here, and other sample data elsewhere, is italicized.

Reply:
The example data is being italicized in the context of figure labels. It's subjective, but I found the un-italicized data to be visually jarring:

![image](https://github.com/amazon-ion/ion-docs/assets/611616/30c1ccc3-d297-4ac3-bde1-5a1e573c92b3)

while the italicized version felt more integrated:

![image](https://github.com/amazon-ion/ion-docs/assets/611616/65ff52ab-4547-48cc-a4dd-d09974647905)